proto/ssn/annotator/v1/annotator.proto

```diff
@@ -155,6 +155,7 @@ message DocumentAnnotatorResponse {
   repeated ssn.type.Candidate ocr_line_nl_payment_id = 22;
   // Return a string containing the text from the document
   string text = 23;
+  string request_id = 24;
```

Comment:
Lets call it `feedback_id` as we talked about the feedback id is not necessarily the `request_id`